src/Deprecated/Engine/Resources/Strings.resx

```diff
@@ -1181,6 +1181,9 @@
   <data name="DefaultTargets" UESanitized="false" Visibility="Public">
     <value>[default]</value>
   </data>
+  <data name="DeprecatedEventSerialization" UESanitized="false" Visibility="Public">
```

Comment:
Where is this being referenced? (shouldn't the [src/Shared/Resources/Strings.shared.resx](https://github.com/dotnet/msbuild/pull/8917/files#diff-c7dca4082a1c8d4a5266ea02ca9e78e6104f37b6039474e9f7b734e972f56f4f) actually be used?)

Reply:
It is in both. I have found resx string "ExpectedEventToBeSerializable" with similar intent in both so I have copied this approach as I believed it might be used from deprecated code too. I believe/hope it will soon be really deprecated and gone.

Reply:
If you're not actually using it in Deprecated, there is no need to add the string (`ExpectedEventToBeSerializable` _is_ used, so that's different). I recommend removing the changes under `src/Deprecated` from the PR.

Reply:
Good point. Removed